docker-setup.sh

```diff
@@ -27,17 +27,21 @@ apt-get install -y tmux
 apt-get install -y gdb valgrind
 
 # install C stuff
-apt-get install -y gcc clang-10 clang-format-10 make
-update-alternatives --install /usr/bin/clang clang /usr/bin/clang-10 100
-update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-10 100
+apt-get install -y gcc clang-16 clang-format-16 make
```

Comment:
Everything looks good, I was wondering if we had to specify the clang version but when I try and update it, it already says it is on the newest version so I am assuming it does not matter.

Reply:
it should just be clang and clang-format